Microsoft.OneWeek.Hack.Microids.IoTDevice.Metadata.Restful/DeviceMetadata.cs

```diff
@@ -0,0 +1,9 @@
+namespace Microsoft.OneWeek.Hack.Microids.IoTDevice.DeviceMetadata.Restful
+{
+    public class DeviceMetadata
+    {
+        public string Id { get; set; }
```

Comment:
I think this class is also duplicated.

Reply:
As discussed :), this class is not present/duplicated in the other project which is based on gRPC as these models are defined in protobuf instead.